row/src/Row.styles.css

```diff
@@ -0,0 +1,6 @@
+dockit-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```

Comment:
not sure it's common to use this. I almost never use this space-evenly.
If you use it. We could also use something like `dockit-row-even`.

Reply:
`space-between` looks more appropriate in this case from the first look, but I have no issue with using `space-evenly` either if it makes sense design wise